### PR TITLE
Added Parallel Park Breadcrumb

### LINKED ISSUE #324: [Feature]Parallel Park Breadcrumb
### Description

Add a parallel park breadcrumb : A showcase of parking cars using breadcrumb designs

### Screenshots

image

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a IWOC'24 contributor.
- [ ] I am a JWOC'24 contributor.
- [ ] I am a SWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CODE_OF_CONDUCT.md)".

### PR DESCRIPTION
**Pull request by AbhineshJha**
# Fixes Issue🛠️

Closes #324 

# Description👨‍💻 

Its a parallel park breadcrumb which shows the functionality of parking car using breadcrumb
# Type of change📄

- [] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How this has been tested✅

# Checklist✅ 

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added demonstration in the form of GIF/video file
- [x] I am an Open Source Contributor

# Screenshots/GIF📷

https://github.com/Rakesh9100/Beautiify/assets/142514166/ae31e94a-79a5-4318-987d-59fa671b121b

### CODE REVIEW

**Comment by Rakesh9100** [comment]
Why html, head, body and some more important tags are missing from your `index.html` file @AbhineshJha

**Comment by AbhineshJha** [comment]
Just learnt!! It is not necessary to add those elements to run the programme..
That's all is metadata for the website optimization..
And there's no need of that .!!

**Comment by Rakesh9100** [comment]
Yes, its metadata and not necessary but one should inclue those tags as they are the base structure of a html file @AbhineshJha

**Comment by AbhineshJha** [comment]
If you wanna add those elements then I'll do that also..!

**Comment by Rakesh9100** [comment]
Yes, do add them everytime as if any beginner or new learner opens the source code then he will get confused @AbhineshJha

**Comment by AbhineshJha** [comment]
Have a review @Rakesh9100

File: Components/Breadcrumbs/Parallel-Park-Breadcrumb/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Parallel Park Breadcrumb</title>
+    <meta name="viewport" content="width=device-width" />
+    <link rel="stylesheet" src="styles.css" />
```
Comment (Rakesh9100) [inline Components/Breadcrumbs/Parallel-Park-Breadcrumb/index.html:7]: It's href and not src @AbhineshJha

**Comment by AbhineshJha** [comment]
Done @Rakesh9100

**Comment by Rakesh9100** [comment]
Nice one, Merged 🥳🎉 @AbhineshJha

**Pull request merged**